amqp-common/src/main/resources/application.yml

```diff
@@ -0,0 +1,7 @@
+amqp:
```

Comment:
we should have separate configuration for mhs queue and pss queue - this is how NME will set it up probably